src/elements/TACSElement.cpp

```diff
@@ -233,9 +245,23 @@ void TACSElement::addAdjResProduct( int elemIndex, double time,
       product += psi[i]*TacsImagPart(tmp[i])/dh;
     }
 #else
-    for ( int i = 0; i < nvars; i++ ){
-      product += psi[i]*(tmp[i] - res[i])/dh;
-    }
+      if (fdOrder < 2){
```

Comment:
Indentation looks off here? 2-spaces only?

Reply:
Should be addressed in last commit